source/deployment/opennebula_installation/frontend_installation.rst

```diff
@@ -158,7 +158,7 @@ The ``/var/lib/one/.one/one_auth`` fill will have been created with a randomly-g
 
     $ echo "oneadmin:mypassword" > ~/.one/one_auth
 
-.. warning:: This will set the oneadmin password on the first boot. From that point, you must use the :ref:`oneuser passwd <manage_users>` command to change oneadmin's password.
+.. warning:: This will set the oneadmin password on the first boot. From that point, you must use the :ref:`oneuser passwd <change_credentials>` command to change oneadmin's password.
```

Comment:
the link should be at the end of the sentence, with the text "More information on how to change oneadmin password here"